spec/views/forms/show_live.html.erb_spec.rb

```diff
@@ -54,12 +56,21 @@
     end
   end
 
-  it "contains a link to declaration" do
-    expect(rendered).to have_link("Declaration", href: "/declaration_path")
+  it "contains declaration" do
+    expect(rendered).to have_css("h2", text: "Declaration")
+    expect(rendered).to have_content(form.declaration_text)
   end
 
-  it "contains a link to what happens next" do
-    expect(rendered).to have_link("What happens next", href: "/what_happens_next_path")
+  context "with no declaration set" do
+    let(:form) { build(:form, :live, :with_pages, id: 2) }
```

Comment:
why do we need to build another form? should we not be setting the existing built forms declaration to nil?